dist/tools/ci/can_fast_ci_run.py

```diff
@@ -183,6 +183,16 @@ def classify_changes(riotbase=None, upstream_branch="master"):
                         help="Use given paths as RIOT's base path (instead of pwd)")
     parser.add_argument("--upstreambranch", default="master",
                         help="The branch / commit containing the upstream state")
+
+    parser.add_argument("--changed-boards", action="store_true",
+                        help="list all boards that should ")
+
+    parser.add_argument("--changed-apps", action="store_true",
+                        help="list all apps that should be built for all boards")
+
+    parser.add_argument("--dump", action="store_true",
```

Comment:
Maybe this is more intuitive?

```suggestion
    parser.add_argument("--json", action="store_true",
```